CapstoneProject/Models/ActivityRecommendation.h

```diff
@@ -6,10 +6,16 @@
 //
 
 #import <Foundation/Foundation.h>
+#import <Parse/Parse.h>
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface ActivityRecommendation : NSObject
+@interface ActivityRecommendation: PFObject <PFSubclassing>
+
+@property (strong, nonatomic) NSString *Activity;
+@property (strong, nonatomic) NSString *EmotionTag;
```

Comment:
The first letter should be the lowercase
```suggestion
@property (strong, nonatomic) NSString *activity;
@property (strong, nonatomic) NSString *emotionTag;
```